components/_buttons.scss

```diff
@@ -0,0 +1,59 @@
+$button-active-color: darken($color-light-blue, 10%);
+$button-line-height: 1.3em;
+
+.wds-button {
+	background-color: $color-light-blue;
+	border: 1px solid $color-light-blue;
+	color: $color-white;
+	cursor: pointer;
+	display: inline-block;
+	font-size: $typescale-size-minus-2;
+	font-weight: 600;
+	letter-spacing: .15px;
+	line-height: $button-line-height;
+	padding: 16px 18px;
+	text-align: center;
+	text-decoration: none;
+	text-transform: uppercase;
+	transition: all 300ms;
+	-webkit-appearance: none;
+	-webkit-border-radius:0;
+
+	&:hover, &.wds-active {
```

Comment:
I would break all separate selectors not only separated by a comma, but additionally with a line break. Same way it's done in `_global-footer.scss`. This way it's easier to distinguish several selectors from a long single selector.